pyproject.toml

```diff
@@ -32,3 +32,19 @@ include = [
 
 [tool.hatch.build.targets.wheel.force-include]
 "./configs" = "sgm/configs"
+
+[tool.hatch.envs.ci]
+# Skip for now, since requirements.txt is used by scripts and includes the project
+# This should be changed when dependencies are handled by Hatch
+skip-install = true 
+
+dependencies = [
+    "pytest"
+]
+
+[tool.hatch.envs.ci.scripts]
+test-inference = [
+    "pip install torch==2.0.1+cu118 torchvision==0.15.2+cu118 torchaudio==2.0.2+cu118 --index-url https://download.pytorch.org/whl/cu118",
+    "pip install -r requirements_pt2.txt",
```

Comment:
Most of these requirements really aren't needed if #44 and #50 get merged – I have identified a minimal requirements set, but that's not much use until those are merged.

Reply:
The current PR has to be merged urgently. But we do not forget your contributions that we should review and merge. Thank you for all of this

Reply:
Indeed, part of the motivation for this PR is to make it easier to test contributions like yours.